src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -6405,6 +6405,65 @@ export class StudyViewPageStore
         );
     }
 
+    @computed get chartMetaSetForClinicalAttributes(): {
+        [id: string]: ChartMeta;
+    } {
+        let _chartMetaSet = _.fromPairs(this._customCharts.toJSON());
+        if (_.isEmpty(this.molecularProfiles.result)) {
```

Comment:
i'm guessing this is some legacy thing but can we put a comment explaining why we delete this?